app/controllers/application_controller.rb

```diff
@@ -107,7 +107,7 @@ def api_deprecation_error(exception = nil)
   def smart_proxy_exception(exception = nil)
     process_error(:redirect => :back, :error_msg => exception.message)
     logger.warn "ProxyAPI operation FAILED: #{exception}"
-    logger.debug exception.backtrace.join("\n")
+    logger.warn exception.backtrace.join("\n")
```

Comment:
That could work.  I'd probably make the description mandatory though so people don't skimp, I can't think of a reason not to include it.  (And probably call it `message`.)

I was looking at logging, and [it seemed that its loggers even supported writing out backtraces](https://github.com/TwP/logging/blob/master/lib/logging/logger.rb#L23).. but it doesn't seem to accept an exception as the second argument for me.